doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-groups-ref.adoc

```diff
@@ -0,0 +1,89 @@
+[id='security-management-rest-api-groups-ref_{context}']
+= Groups
+
+The Security Management REST API supports the following endpoints for managing groups in {CENTRAL}. The Security Management REST API base URL is `\http://SERVER:PORT/{URL_COMPONENT_CENTRAL}/rest/`. All requests require HTTP Basic authentication or token-based authentication for the `admin` user role.
+
+[GET] /groups::
++
+--
+Returns all groups in {CENTRAL}.
+
+.Example server response (JSON)
+[source,json]
+----
+[{
```

Comment:
This left curly bracket should probably be on the next line to have the same formatting as on other places?

Reply:
Done